### PR TITLE
[HOPSWORKS-3278] add type to feature view dto

### PR DESCRIPTION
**Pull request by ErmiasG**

### CODE REVIEW

**Comment by ErmiasG** [comment]
https://github.com/logicalclocks/hops-testing-ee/pull/544

**Review by moritzmeister — APPROVED** [review]

File: python/hsfs/client/exceptions.py [hunk 1]
```diff
@@ -19,14 +19,18 @@ class RestAPIError(Exception):
     """REST Exception encapsulating the response object and url."""
 
     def __init__(self, url, response):
-        error_object = response.json()
+        try:
+            error_object = response.json()
```
Comment (moritzmeister) [inline python/hsfs/client/exceptions.py:23]: why was this needed?

Comment (ErmiasG) [reply python/hsfs/client/exceptions.py:23]: B/c it was failing if there is no body and not showing the error. Hsml does the same.

**Review by moritzmeister — APPROVED** [review]
